CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ We appreciate your help!
 
 If you are unsure whether you have found a bug, please consider searching existing issues in github and asking in Ballerina-Dev Google Group.
 
-> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). See the security policy for details.
+> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). For details, see the [security policy](https://ballerina.io/security/)].
```

Comment:
IMO, we should include the same link from https://github.com/ballerina-platform/ballerina-dev-website/blob/master/community.md under "Want to Report Issues" too.

Reply:
Added this via https://github.com/ballerina-platform/ballerina-dev-website/pull/131/commits/478371da91238165498cb79c72d29b75cfd2cb02.